pkgs/top-level/python-packages.nix

```diff
@@ -15483,6 +15483,17 @@ self: super: with self; {
 
   tree-sitter0_21 = callPackage ../development/python-modules/tree-sitter0_21 { };
 
+  tree-sitter-grammars = lib.recurseIntoAttrs (lib.mapAttrs
+    (name: grammarDrv: callPackage ../development/python-modules/tree-sitter-grammars { inherit name grammarDrv; })
+    (lib.attrsets.removeAttrs pkgs.tree-sitter.builtGrammars [
+      # ImportError: /nix/store/7w7piy6hpdj1swdg5r0bz47gk2g3q855-tree-sitter-perl-grammar-0.22.5/parser: undefined symbol: isnumber
+      "tree-sitter-perl"
+      # ImportError: /nix/store/6rkhjf2mhwnfqwq2962fvv9bnd4xmpk7-python3.11-python-tree-sitter-ql-dbscheme-0.22.5/lib/python3.11/site-packages/tree_sitter_ql_dbscheme/_binding.abi3.so: undefined symbol: tree_sitter_ql_dbscheme
+      "tree-sitter-ql-dbscheme"
+      # ImportError: /nix/store/ybmnykhrgj6sghw4r7ypf897c4wrncr5-python3.11-python-tree-sitter-org-nvim-0.22.5/lib/python3.11/site-packages/tree_sitter_org_nvim/_binding.abi3.so: undefined symbol: tree_sitter_org_nvim
+      "tree-sitter-org-nvim"
```

Comment:
Each of these might be resolved by simply bumping its source.

Some could possibly be resolved with some tinkering, which we decided should not delay the submitting of this PR for review.

Reply:
Resolved. All grammars pass!